### ISSUE TITLE
[BUG] Alignment for tiddler icons should be improved

### ISSUE THREAD

**Issue by pmario** [Jermolene/TiddlyWiki5#6258]
As seen in the screenshot the icons are some pixels off to the tiddler title heading. 

IMO it should look good with titles like "Hello There" and also titles like "hello-there"

![image](https://user-images.githubusercontent.com/374655/142768272-e65fcc08-3dad-424c-8e9e-e0688fb4896c.png)

This issue isn't critical, but it also shouldn't be forgotten.

**Comment by pmario** [comment]
improved screenshot

![image](https://user-images.githubusercontent.com/374655/142768453-a1db61c1-6b45-4ecf-9c5e-8ce03ed4ea09.png)

**Comment by kookma** [comment]
@pmario 
I use the cascades to set tiddler icon, it seems I have not the same issue! but here the same svgs as core svg is used!
I have not investigated the css to see how icon is aligned!
![img_692_chrome](https://user-images.githubusercontent.com/830394/142771853-8bf9a83f-4964-497e-b566-eed5bd1e203b.png)
![img_693_chrome](https://user-images.githubusercontent.com/830394/142771854-b165e743-04b3-4769-8791-8eb78a13b290.png)
![img_695_chrome](https://user-images.githubusercontent.com/830394/142771855-83001159-4080-488b-8302-ad8b3b27c72f.png)
![img_697_chrome](https://user-images.githubusercontent.com/830394/142771856-883946e0-87c7-4a83-9fea-10c9dad70d94.png)
![img_698_chrome](https://user-images.githubusercontent.com/830394/142771858-7987834f-ab04-4653-b565-8bb1fafc5ee6.png)

**Comment by pmario** [comment]
See: 
![image](https://user-images.githubusercontent.com/374655/142772356-6566cfdf-0ae8-496b-9145-bec8e658a102.png)

**Comment by pmario** [comment]
They look fine: ... BUT I used a 160% browser zoom to make the images bigger on my scree ... So it's all realtive

![image](https://user-images.githubusercontent.com/374655/142772406-94c38e3b-606b-4dbf-a38e-72068c0e0bf9.png)

**Comment by pmario** [comment]
>I have not investigated the css to see how icon is aligned!

Yes, that's probably the first element we have to have a closer look

**Comment by Jermolene** [comment]
It would be good to fix this. I recommend using `$:/core/images/picture` as the test image because it 100% fills its boundaries, unlike some icons which incorporate a margin.

**Comment by kookma** [comment]
> See: The bulb seems to have a little problem

Yes, when zooming, it appears, the alignment is not correct! By the way I myself like middle align, I am not sure what TW prefers?

**Comment by pmario** [comment]
@kookma It is vertically aligned middle, but to a DOM element where it has no meaning. We probably need to change the DOM structure a bit and make it valid first. 

```
<span>
    <span class="tc-tiddler-title-icon" style="fill:#1e90ff;">
        <svg width="22pt" height="22pt" class="tc-image-globe tc-image-button" viewBox="0 0 128 128">
            <g fill-rule="evenodd"><path d="M72.811 ... snip ... 8 64s25.072 56 56 56z"></path></g>
        </svg>
    </span>
    <h2 class="tc-title">HelloThere</h2>
</span>
```

At the moment the structure looks like this. ... SPANs need to contain [phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#phrasing_content) which H2 is _not_ .... Click the link an scroll a bit up. ... 

Since the ViewTemplates have been changed for 5.2.1 already, I think it's a good possibility to check this too.

**Comment by kookma** [comment]
> Since the ViewTemplates have been changed for 5.2.1 already, I think it's a good possibility to check this too.

Quite agree! It worths fixing!
